app/client/tests/unit/store/index.spec.ts

```diff
@@ -0,0 +1,19 @@
+import index from '@/store/index';
```

Comment:
I'm guessing these are just to keep codecov happy? I suppose rather than testing the mutation names etc it might be quicker to just expect mutations toBe the mutations imported from the mutations file?